src/app/shared/functions/paths.ts

```diff
@@ -0,0 +1,15 @@
+
+// entityKey sport
+// pathArray [ 'sport', 'rugby', 'superRugby', 'wallabies' ]
+
+export const getBreadCrumbPath = (entityKey: string, pathArray: string[]) => {
+  if (!entityKey || !pathArray?.length) return '#'
```

Comment:
i didn't really understand this, what is the implication of returning `#`?

Reply:
was just returning the root so if there was no link would return the root but probably should just return an empty string and this no link, have updated.